src/search.ts

```diff
@@ -99,8 +100,53 @@ function getRequiredModules(requireNodes: Node[]): SearchValue {
       // If expression is not a literal or a template literal, it is dynamically
       // evaluated
     } else {
-      dynamicEvalPos.push(pos);
+      dynamicArgPos.push(pos);
+    }
+  });
+  return {requiredModules, dynamicArgs: dynamicArgPos, dynamicRequire: []};
+}
+
+/**
+ * Returns list of positions where there are dynamic require calls
```

Comment:
Can you elaborate in this comment on what constitutes a "dynamic require call"?

Reply:
In addition, we should think about if "dynamic require call" is the best term to use here. To differentiate from dynamic require arguments which are arguments that are only known at runtime, maybe we should call this scenario something else.